_docs/geyser/translations.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Supported languages & translations
+---
+
+Geyser supports a wide variety of languages to offer the best experience for players all over the globe.
+When players join your server, Geyser will automatically detect their locale and provide the appropriate language.
+
+## What languages are supported?
+We aim to support any of the Bedrock languages - below is a list of them and all the language codes. 
+
+|Name                        |Code |
+|----------------------------|-----|
+|Bulgarian                   |bg_bg|
+|Czech                       |cs_cz|
+|Danish                      |da_dk|
+|German                      |de_de|
+|Greek                       |el_gr|
+|British English             |en_gb|
+|American English            |en_us|
+|Spanish                     |es_es|
+|Mexican Spanish             |es_mx|
+|Finnish                     |fi_fi|
+|Canadian French             |fr_ca|
+|French                      |fr_fr|
+|Hungarian                   |hu_hu|
+|Indonesian                  |id_id|
+|Italian                     |it_it|
+|Japanese                    |ja_jp|
+|Korean                      |ko_kr|
+|Dutch                       |nl_nl|
+|Norwegian Bokmål            |nb_no|
+|Polish                      |pl_pl|
+|Brazilian Portuguese        |pt_br|
+|Portuguese                  |pt_pt|
+|Russian                     |ru_ru|
+|Slovak                      |sk_sk|
+|Swedish                     |sv_se|
+|Turkish                     |tr_tr|
+|Ukrainian                   |uk_ua|
+|Chinese Simplified (China)  |zh_cn|
+|Chinese Traditional (Taiwan)|zh_tw|
+
+### Additional languages
+These languages are also supported, and are available in the Bedrock client using [this](https://www.curseforge.com/minecraft-bedrock/addons/translations-for-minecraft) resource pack.
```

Comment:
I know this is getting really, uh, niche? What's the word for it. Anyway, we should tell the user what they're clicking on instead of just saying "this"

```suggestion
These languages are also supported, and are available in the Bedrock client using [the Translations for Minecraft resource pack](https://www.curseforge.com/minecraft-bedrock/addons/translations-for-minecraft).
```

Reply:
Sure, not a bad idea